frontend/src/app/admin/page.tsx

```diff
@@ -5,8 +5,16 @@ import AdminContent from "@/components/AdminContent/AdminContent";
 import { getServerSession } from "next-auth";
 import { authOptions } from "@/lib/auth";
 import navbar from "@/assets/navbar.svg";
+import { Metadata } from "next";
 
-export default async function () {
+export async function generateMetadata(): Promise<Metadata> {
```

Comment:
I think we can just do export const metadata = {} here since there is no data fetching